docs/agents/sematext-agent/containers/installation.md

```diff
@@ -91,3 +93,5 @@ docker service create --mode global --name st-agent \
 -e REGION=<US or EU> \
 sematext/agent:latest
 ```
+
+_[Read](../permission-requirements.md#container-setups) more about why Sematext Agent needs access to host' files and directories._
```

Comment:
_[Read](../permission-requirements.md#container-setups) more about why Sematext Agent needs access to host' files and directories._

=>

_[Read more](../permission-requirements.md#container-setups) about why Sematext Agent needs access to host files and directories._